src/include/cluster/dialogs.rb

```diff
@@ -708,6 +708,205 @@ def SaveSecurity
       nil
     end
 
+    def ValidateCorosyncQdevice
+      if UI.QueryWidget(Id(:corosync_qdevice), :Value) == false
+        return true
+      end
+
+      if UI.QueryWidget(Id(:qdevice_model), :Value) != "net"
+        Popup.Message(_("The model basically defines the type of arbitrator, currently only net is support"))
+        UI.SetFocus(:qdevice_model)
+        return false
+      end
+
+      if UI.QueryWidget(Id(:qdevice_votes), :Value).to_i <= 0
+        Popup.Message(_("Qdevice votes must be a positive integer"))
+        UI.SetFocus(:qdevice_votes)
+        return false
+      end
+
+      if !IP.Check(UI.QueryWidget(Id(:qdevice_host), :Value))
+        Popup.Message(_("Qdevice host mush have a valid IP address"))
+        UI.SetFocus(:qdevice_host)
+        return false
+      end
+
+      if UI.QueryWidget(Id(:qdevice_port), :Value).to_i <= 0
+        Popup.Message(_("The corosync qdevice port must be a positive integer"))
+        UI.SetFocus(Id(:qdevice_port))
+        return false
+      end
+
+      if !["ffsplit", "lms", "test", "2nodelms"].include?(UI.QueryWidget(Id(:qdevice_algorithm), :Value))
+        Popup.Message(_("The algorithm only can be one of the ffsplit, lms, test or 2nodelms.
+          YaST will overwrite test and 2nodelms."))
+        UI.SetFocus(Id(:algorithm))
+        return false
+      end
+
+      if !["lowest", "highest"].include?(UI.QueryWidget(Id(:qdevice_tie_breaker), :Value)) &&
+        (UI.QueryWidget(Id(:tie_breaker), :Value).to_i <= 0)
+        Popup.Message(_("The tie breaker can be one of lowest, highest or a valid node id (number)"))
+        UI.SetFocus(Id(:tie_breaker))
+        return false
+      end
+
+      true
+    end
+
+    def SaveCorosyncQdevice
+      Cluster.corosync_qdevice = Convert.to_boolean(UI.QueryWidget(Id(:corosync_qdevice), :Value))
+
+      Cluster.qdevice_model = UI.QueryWidget(Id(:qdevice_model), :Value)
+      Cluster.qdevice_votes = UI.QueryWidget(Id(:qdevice_votes), :Value).to_s
+
+      Cluster.qdevice_host = UI.QueryWidget(Id(:qdevice_host), :Value)
+      Cluster.qdevice_port = UI.QueryWidget(Id(:qdevice_port), :Value).to_s
+      Cluster.qdevice_tls = UI.QueryWidget(Id(:qdevice_tls), :Value)
+      Cluster.qdevice_algorithm = UI.QueryWidget(Id(:qdevice_algorithm), :Value)
+      Cluster.qdevice_tie_breaker = UI.QueryWidget(Id(:qdevice_tie_breaker), :Value)
```

Comment:
and here `qdevice_tie_breaker` is stored and not without prefix.

Reply:
Yes, should change all :tie_breaker to :qdevice_tie_breaker.